onnxruntime/core/providers/cpu/quantization/conv_integer.cc

```diff
@@ -42,12 +54,12 @@ Status ConvInteger::Compute(OpKernelContext* context) const {
   if (num_inputs >= 3) {
     const auto* X_Zero_Point = context->Input<Tensor>(2);
     ORT_ENFORCE(IsScalarOr1ElementVector(X_Zero_Point), "Must be a scalar or 1D tensor or size 1.");
-    input_offset = *(X_Zero_Point->Data<uint8_t>());
+    input_offset = *static_cast<const uint8_t*>(X_Zero_Point->DataRaw());
```

Comment:
it is not right to just convert the data type. you need to specify if BIsSigned in the MLAS_GEMM_QUANT_SHAPE_PARAMS  below.